front/app/modules/commercial/id_bogus/components/VerificationFormBogus.tsx

```diff
@@ -56,10 +60,9 @@ const VerificationFormBogus = memo<Props>(
         try {
           await verifyBogus(desiredError);
 
-          const endpointsToRefetch = [
-            `${API_PATH}/users/me`,
-            `${API_PATH}/projects`,
-          ];
+          queryClient.invalidateQueries({ queryKey: projectsKeys.all() });
```

Comment:
This is also about the action descriptors, correct? I am just wondering if we could get away with invalidating all `items` and keeping the list cache. But maybe not worth the optimization 🤔